test/support/utils.rb

```diff
@@ -53,11 +57,28 @@ def create_pseudo_terminal(boot_options: "-r debug/run")
 
       timeout_sec = (ENV['RUBY_DEBUG_TIMEOUT_SEC'] || 10).to_i
 
-      PTY.spawn("#{RUBY} #{boot_options} #{temp_file_path}") do |read, write, pid|
+      PTY.spawn("sh") do |read, write, pid|
+        write.puts("export PS1='Hello Prompt\n'")
+        if boot_options == "-r debug/open"
```

Comment:
I think the remote server can be started with just one line? like https://github.com/ruby/debug/pull/75/files#diff-ee14bf65f14927f7778a246f90c5cec0c581be3e2658c2a8a3726c59db5be3f9R43

Reply:
Yeah, I will try it.